### PR TITLE
testSongLyricsText()

### PR DESCRIPTION
**Pull request by ElNov686**
class Song99BottlesElenaNov686Test added, 
testSongLyricsText() created

### CODE REVIEW

File: src/test/java/qa_java_beginners/Song99BottlesElenaNov686Test.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            pTexts[i] = getDriver().findElement(By.xpath("//body/div[@id='wrap']/div[@id='main']/p[" + index + "]"))
+                    .getText();
+        }
+
+        String actualResult = "";
+        for (int i = 0; i < pTexts.length; i++){
+            actualResult = actualResult + pTexts[i];
+        }
+
+
```
Comment (jbioshock1912) [inline src/test/java/qa_java_beginners/Song99BottlesElenaNov686Test.java:132]: пустые строчки

**Review by jbioshock1912 — APPROVED** [review]

File: src/test/java/qa_java_beginners/Song99BottlesElenaNov686Test.java (hunk 1 above)
Comment (ElNov686) [reply src/test/java/qa_java_beginners/Song99BottlesElenaNov686Test.java:132]: Спасибо!

File: src/test/java/qa_java_beginners/Song99BottlesElenaNov686Test.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Song99BottlesElenaNov686Test.java:133]: Lots of empty lines at the end. You need to delete them.

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/qa_java_beginners/Song99BottlesElenaNov686Test.java (hunk 2 above)
Comment (ElNov686) [reply src/test/java/qa_java_beginners/Song99BottlesElenaNov686Test.java:133]: Thank you!

**Pull request merged**